### PR TITLE
Makes Genetics less miserable + Removes mentions of midget from code.

### PR DESCRIPTION
**Pull request by ItsMarmite**
## What Does This PR Do

image

- Replaces instances of midget in the code with dwarf.

- Tweaks DNA modifier access consoles to generate injectors with a 10 second cooldown instead of a 30 seconds cooldown.

- The genes Hulk, No Breathing Telekinesis, Cloak of Darkness and Chameleon have had their random chance to activate on injection removed, because injecting yourself 20 times for a gene to activate is about as fun as a bread sandwich.

- Genetic instability has had its damage numbers doubled to make it deadlier to handle, requiring those who want to combine multiple powerful genes together to dabble in disabilities.

Hallucination now gives Major tier gene stability
Clumsiness now gives Moderate tier gene stability
Wingdings now gives Moderate tier gene stability
Dizziness now gives Moderate tier gene stability
Colourblindness now gives Minor tier gene stability
Mute now gives Major tier gene stability.
Obesity no longer gives gene stability

No Breathing now gives Major tier gene instability
Chameleon and Cloak of Darkness now gives Major tier gene instability.
No Prints now gives Moderate tier gene instability
Dwarf now gives Moderate tier gene instability
Matter Eater now gives Moderate tier gene instability
Jumpy now gives Moderate tier gene instability
Polymorph now gives Moderate tier gene instability
Flash protection now gives Moderate tier gene instability.
Sober now gives Minor tier gene instability
Psi Resist now gives Minor tier gene instability

## Why It's Good For The Game
Midget is generally considered an offensive term and we probably shouldn't be using it, especially when the gene itself is known as Dwarf. Genetics injectors are miserably slow to print out and artificially extend the time it takes to get genes as a geneticist leading to long periods of a round where you sit there half afk printing injectors with several blocks being a guess of whether they're going to actually contain a gene. If you're lucky enough that it isn't a disability, it could be nothing or it could also be a superpower. Such a superpower could be activated if you're lucky enough, and luck is a boring thing to balance the enjoyment of a job around.

I'm hoping this will encourage genetics to get the work done in the first half of the round roughly if they work together, and then they can actually mess around with the genes, making custom injector combinations and allowing more of the crew to have fun being superpowered.

If everyone is super, no one is.

## Testing
I had to actually play genetics. It was slightly less awful.

## Changelog
:cl:
tweak: Removes instances of midget from the codebase.
tweak: Several genes and disabilities have had their instability categories altered.
tweak: Activation chance has been removed from a number of genetic powers.
tweak: Genetic instability is now deadlier.
tweak: Genetics injectors are now faster to print.
/:cl:

### CODE REVIEW

File: code/game/dna/mutations/mutation_powers.dm [hunk 1]
```diff
@@ -6,9 +6,8 @@
 	name = "No Breathing"
 	activation_messages = list("You feel no need to breathe.")
 	deactivation_messages = list("You feel the need to breathe, once more.")
-	instability = GENE_INSTABILITY_MODERATE
+	instability = GENE_INSTABILITY_MAJOR
 	traits_to_add = list(TRAIT_NOBREATH)
-	activation_prob = 25
```
Comment (lewcc) [inline code/game/dna/mutations/mutation_powers.dm:11]: Can you just remove this var from the supertype entirely, especially if it's now unused?

**Comment by Ionward** [comment]
thank you, finally, genetics was a department of random suffering and this should help loads. it was honestly faster to not have a second geneticist so you could use the 2nd console to print a clean SE after testing one.

File: code/game/dna/mutations/mutation_powers.dm (hunk 1 above)
Comment (ItsMarmite) [reply code/game/dna/mutations/mutation_powers.dm:11]: Pushed.

**Comment by ItsMarmite** [comment]
> Isnt Matter Eater free infinite freedom implant? Why only Moderate tier?

Because its currently a minor, and I don't feel its on par with stun immunity, a changeling power and a vampire power ultimately. It can eat cuffs sure but its also blocked by a mask.

File: code/game/dna/dna2_domutcheck.dm [hunk 2]
```diff
@@ -31,11 +31,6 @@
 	// Current state
 	var/mutation_active = M.dna.GetSEState(mutation.block)
 
-	// Sanity checks, don't skip.
-	if(mutation_active && !mutation.can_activate(M, flags))
-		//testing("[M] - Failed to activate [gene.name] (can_activate fail).")
-		return FALSE
-
```
Comment (SteelSlayer) [inline code/game/dna/dna2_domutcheck.dm:38]: Why remove this?

**Comment by tipoima** [comment]
> Sober now gives Minor tier gene instability

Maybe then remove it from Detectives? Spawning in with instability for a "haha alcoholic detective" joke seems annoying.

File: code/game/dna/dna2_domutcheck.dm (hunk 2 above)
Comment (SteelSlayer) [reply code/game/dna/dna2_domutcheck.dm:38]: Yes please.

**Comment by Ionward** [comment]
> > Sober now gives Minor tier gene instability
> 
> Maybe then remove it from Detectives? Spawning in with instability for a "haha alcoholic detective" joke seems annoying.

instability in this case refers to the point values of a genetic block being positive or negative, there's thresholds to it that are not changed in this (just the damage once you reach those thresholds). the only thing this will affect is if a detective wants more powers they need to either take a disability or get the sober gene removed (or take less powers)

**Review by lewcc — CHANGES_REQUESTED** [review]
Please address steel's note, otherwise looks pretty good to me.

File: code/game/dna/dna2_domutcheck.dm (hunk 2 above)
Comment (ItsMarmite) [reply code/game/dna/dna2_domutcheck.dm]: Pushed.

**Comment by ItsMarmite** [comment]
> Please address steel's note, otherwise looks pretty good to me.

Change pushed.

File: code/game/dna/mutations/_mutations.dm [hunk 3]
```diff
@@ -10,8 +10,6 @@
 	/// Trait to give, if any
 	var/traits_to_add = list()
 	/// Activation probability
-	var/activation_prob = 100
-	/// Possible activation messages
 	var/list/activation_messages = list()
```
Comment (Henri215) [inline code/game/dna/mutations/_mutations.dm:13]:
```suggestion
	/// Possible activation messages
	var/list/activation_messages = list()
```
You removed the wrong comment

File: code/game/dna/mutations/_mutations.dm [hunk 4]
```diff
@@ -10,8 +10,6 @@
 	/// Trait to give, if any
 	var/traits_to_add = list()
 	/// Activation probability
```
Comment (Henri215) [inline code/game/dna/mutations/_mutations.dm:12]:
```suggestion
```

File: code/game/dna/mutations/disabilities.dm [hunk 5]
```diff
@@ -241,7 +241,7 @@
 	desc = "Completely shuts down the speech center of the subject's brain."
 	activation_messages = list("You feel unable to express yourself at all.")
 	deactivation_messages = list("You feel able to speak freely again.")
-	instability = -GENE_INSTABILITY_MODERATE
+	instability = -GENE_INSTABILITY_MAJOR
```
Comment (Henri215) [inline code/game/dna/mutations/disabilities.dm:244]: Mute missing from the PR description i think

Comment (ItsMarmite) [reply code/game/dna/mutations/disabilities.dm:244]: Good catch, I'll add it in must've missed it in the big list.

File: code/game/dna/mutations/_mutations.dm [hunk 6]
```diff
@@ -26,8 +24,6 @@
 /datum/mutation/proc/can_activate(mob/M, flags)
 	if(flags & MUTCHK_FORCED)
 		return TRUE
-	// Probability check
-	return prob(activation_prob)
```
Comment (Henri215) [inline code/game/dna/mutations/_mutations.dm:30]: Since there is no more RNG involved you want to make this proc always return TRUE at the end?

Comment (Oyu07) [reply code/game/dna/mutations/_mutations.dm:30]: If we are returning TRUE always, is there even a need for the MUTCH_FORCED check?
cant it be entirely
```
/datum/mutation/proc/can_activate(mob/M, flags)
     return TRUE
```
?

Comment (ItsMarmite) [reply code/game/dna/mutations/_mutations.dm:30]: I'll try this out later tonight and see if it causes any issues.

Comment (ItsMarmite) [reply code/game/dna/mutations/_mutations.dm:30]: Tested out Oyu's suggested change, everything seems to work as intended.

**Review by Henri215 — APPROVED** [review]
Looks good to marm

**Review by Oyu07 — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**